.github/workflows/main.yml

```diff
@@ -38,6 +38,11 @@ jobs:
             rust: stable
             target: aarch64-apple-ios
             no_run: --no-run
+          - build: aarch64-tvos
+            os: macos-latest
+            rust: nightly
+            target: aarch64-apple-tvos
```

Comment:
There's no pre-built libstd for this target, so rustup failed in CI.

IMO it's better to create a new job since it woukd have to use `-Zbuild-std` to manually compile the libstd.